docs/how-tos/provision.md

```diff
@@ -0,0 +1,310 @@
+---
+title: "Configure provisioning with viam-agent"
+linkTitle: "Prepare provisioning"
+weight: 68
+type: "docs"
+description: "Install and configure viam-agent for provisioning one or many machines."
+images: ["/installation/thumbnails/install.png"]
+imageAlt: "Install Viam"
+---
+
+The provisioning subsystem is a feature of `viam-agent`, which you can install as part of your manufacturing process. `agent-provisioning` will then perform the rest of the first-time setup for your machine once an end user sets up the machine.
+
+This is useful when deploying a fleet of machines directly from the factory to a customer, or when bundling proprietary software on your Viam machine.
+
+This guide will show you how to install and configure `viam-agent`.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Configure `agent-provisioning`](#configure-agent-provisioning)
+1. [Decide on the provisioning method](#decide-on-the-provisioning-method)
+1. [Install `viam-agent`](#install-viam-agent)
+
+{{< /alert >}}
+
+## Prerequisites
+
+{{% expand "One or more physical devices with supported operating system" %}}
+
+To find out more about supported systems, see [Compatibility](/installation/#compatibility).
+
+{{% /expand%}}
+
+{{% expand "A machine fragment" %}}
```

Comment:
This is only required when using the mobile app. It's optional for the captive portal method (the portal will only display it, not do anything with it.)